policies/postgres/init.go

```diff
@@ -51,6 +51,7 @@ func migrateDB(db *sqlx.DB) error {
 						orb_tags       JSONB NOT NULL DEFAULT '{}',
 
 						backend        TEXT NOT NULL,
+                        schema_version TEXT NOT NULL,
```

Comment:
So for the first time we cannot just update "policies_1", we have to make a migration here with "policies_2". See this for an example https://github.com/mainflux/mainflux/blob/aa014c21915b42764e070276a1e05c49a43feac7/things/postgres/init.go#L83